Task_1/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Task_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Class class1 = new Class(new BadPupil("igor"), new ExcelentPupil());
```

Comment:
Use "var" if possible
```suggestion
            var class1 = new Class(new BadPupil("igor"), new ExcelentPupil());
```